cmd/openshift/operator/kodata/tekton-addon/1.4.0/addons/02-clustertasks/git-clone/git-clone-v0-22-0-task.yaml

```diff
@@ -1,16 +1,16 @@
 # auto generated by script/update-tasks.sh
 # DO NOT EDIT: use the script instead
-# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.19/task/git-clone/0.2/git-clone.yaml
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.22/task/git-clone/0.3/git-clone.yaml
 #
 ---
 apiVersion: tekton.dev/v1beta1
 kind: ClusterTask
 metadata:
-  name: git-clone-v0-19-0
+  name: git-clone-v0-22-0
   labels:
-    app.kubernetes.io/version: "0.2"
+    app.kubernetes.io/version: "0.3"
   annotations:
-    tekton.dev/pipelines.minVersion: "0.19"
+    tekton.dev/pipelines.minVersion: "0.21.0"
```

Comment:
1. shouldn't be 0.22?
2. we should unify the format of this version (sometimes we use the third number, sometimes not)